source/2016-08-25-5-rules-to-git-better.html.markdown

```diff
@@ -0,0 +1,112 @@
+---
+title: 5 rules to git better
+date: 2016-08-25 19:41 UTC
+author: lex
+tags:
+---
+
+Git is one the greatest developer tools available. But using git in a wrong way
```

Comment:
"one **of** the"

Ausserdem würde ich Git konsequent gross schreiben (scheint die offizielle Schreibweise zu sein), auch im Titel.